app/fruits/page.tsx

```diff
@@ -1,43 +1,40 @@
-import { cookies } from 'next/headers';
 import Link from 'next/link';
 import { fruits } from '../../database/fruits';
-import { CookieValue } from '../../util/cookies';
+import { getCookieByName } from '../../util/cookies';
 
 export const metadata = {
   title: 'Fruits',
   description: 'This is my Fruits Page',
 };
 
+// we add this only if we have no dynamic function as cookies or headers
+export const dynamic = 'force-dynamic';
+export const revalidate = false;
```

Comment:
- do we need both of these?
- isn't this handled already by the call to `cookies()` in the `util/cookies` file? (because it's being imported in this file)